CONTRIBUTING.md

```diff
@@ -117,7 +117,35 @@ further information on making changes and committing snapshots.*)
 `Modules/ThirdParty` directory, please read our
 [UpdatingThirdParty](Documentation/UpdatingThirdParty.md) guide.*
 
-Standard prefixes for ITK commit messages:
+Making Breaking Changes
+-----------------------
+
+Breaking changes are defined in ITK as those changes that introduce changes to
+the API of the immediately previous release version, and as such, make a
```

Comment:
*the immediately previous release version* -> *the [major version](https://semver.org/) of the toolkit*

Reply:
Done in 56ed51a.